view-user/src/components/common/Header/Header.tsx

```diff
@@ -1,24 +1,30 @@
 import React, { ReactNode } from "react";
 import styles from "./Header.module.css";
 import { useRouter } from "next/router";
+import { IoHelpCircleOutline } from "react-icons/io5";
 
-interface HeaderProps {
-  children: ReactNode;
-  user: string;
-}
-
-const Header = ({ children, user }: HeaderProps) => {
+const Header = () => {
   const router = useRouter();
   return (
     <div className={styles.container}>
       <div className={styles.main}>
-        <div className={styles.title} onClick={() => router.push("/")}>
-          <p>NUTFES BINGO {user}</p>
-        </div>
-        <div>{children}</div>
+        <img
```

Comment:
```suggestion
        <img
        onClick={() => router.push("/")}
```
imgタグにonClickあったか忘れたけどこんな感じ